src/app/view/components/login/dot-login-component/login-container.component.ts

```diff
@@ -81,21 +79,7 @@ export class LoginContainerComponent {
             );
     }
 
-    /**
-     * Display the forgot password card
-     */
-    showForgotPassword(): void {
-        this.dotRouterService.goToForgotPassword();
-    }
```

Comment:
why was this move to the component?

Reply:
to make it directly in the component and not in the container